cmd/bucket-meta.go

```diff
@@ -115,13 +115,17 @@ func (b *bucketMetadata) convertLegacyLockconfig(ctx context.Context, objectAPI
 			return err
 		}
 
-		logger.LogIf(ctx, deleteConfig(ctx, objectAPI, configFile))
+		if err := deleteConfig(ctx, objectAPI, configFile); err != nil {
+			if !errors.Is(err, errConfigNotFound) {
```

Comment:
inner `if` can be merged with the above.

Reply:
sure